### ISSUE TITLE
Can macOS be supported? I noticed that the software was developed using QT.

### ISSUE THREAD

**Issue by xavierchen2021** [AntiMicroX/antimicrox#455]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe.

可以支持macOS吗？我注意到软件是使用QT开发的
Can macOS be supported? I noticed that the software was developed using QT.

**Comment by pktiuk** [comment]
Yes, it could be supported, but it would require a macos developer.

In the past there were some attempts to ass macOS suppport, but they were never finished https://github.com/AntiMicro/antimicro/pull/257

**Comment by pktiuk** [comment]
I created a new branch with changes introduced by @fwcd to not lose them after archiving the legacy repo.  
 https://github.com/AntiMicroX/antimicrox/tree/macos_attempt
